sdk/src/keystore/error.rs

```diff
@@ -1,4 +1,8 @@
 //! Keystore Errors
+#[cfg(not(any(feature = "std", feature = "wasm")))]
+use alloc::string::String;
+#[cfg(any(feature = "std", feature = "wasm"))]
+use std::string::String;
```

Comment:
FYI, you can just use `alloc` for both `std` and `no_std` builds. But that's kind of a nit, I can fix that later.